generator/gen_bindings.go

```diff
@@ -546,16 +546,8 @@ func (gen *Generator) proxyArgFromGo(memTip tl.Tip, name string,
 		proxy = fmt.Sprintf("%s(%s)", helper.Name, name)
 		return proxy, helper.Nillable
 	case isPlain && goSpec.Slices != 0: // ex: []byte
-		gen.submitHelper(sliceHeader)
-		if goSpec.Base == "unsafe.Pointer" &&
-			(len(goSpec.Raw) == 0 || goSpec.Raw == "unsafe.Pointer") {
-			// Go 1.8+
-			cgoSpec.Base = "unsafe.Pointer"
-		}
-		helper := gen.getCopyBytesHelper(cgoSpec)
-		gen.submitHelper(helper)
-		proxy = fmt.Sprintf("%s((*sliceHeader)(unsafe.Pointer(&%s)))", helper.Name, name)
-		return proxy, helper.Nillable
```

Comment:
why don't we need to return the proxy and Nillable anymore?

Reply:
you return proxy by default - thats what an empty return statement does. I'm actually not sure what the default return value for it is in our case. In general, I used the diff between uldaq.go of the standard c-for-go translation + my hand edits and the uldaq.go that's generated by this modified c-for-go as a way to diagnose my changes